src/components/navigation/RouterLink.tsx

```diff
@@ -0,0 +1,26 @@
+import { forwardRef } from 'react';
+import { NavLink as BaseNavLink, NavLinkProps } from 'react-router-dom';
+
+const RouterLink = forwardRef<HTMLAnchorElement, NavLinkProps>(
+    function NavLinkRef(refProps, ref) {
+        const activeClassName = 'Mui-selected';
```

Comment:
Is `activeClassName` needed?

Reply:
I left it in since before it was adding two classes and it's easier to just leave it as it.